setup.py

```diff
@@ -524,12 +524,10 @@ def maybe_cythonize(extensions, *args, **kwargs):
     """
     Render tempita templates before calling cythonize
     """
-    if len(sys.argv) > 1 and "clean" in sys.argv:
+    if "clean" in sys.argv or "sdist" in sys.argv:
```

Comment:
can you update the comment here to include this is to not include cythonized files in the sdist as well?